client/lib/analytics/test/utils.js

```diff
@@ -0,0 +1,34 @@
+/** @format */
+/**
+ * External dependencies
+ */
+
+import { expect } from 'chai';
```

Comment:
Use jest! `expect` is available without any import. We'll get better error messages too